src/components/StudyForm/StudyForm.tsx

```diff
@@ -257,11 +257,41 @@ const StudyForm: React.FC<StudyFormProps> = ({
                 data-tour={`${TourId.StudyForm}-2`}
                 className={`${(fieldState.isTouched || formState.isSubmitted) && "ion-touched"} ${fieldState.invalid && "ion-invalid"}`}
                 labelPlacement="floating"
+                multiple={Array.isArray(field.value)}
                 onIonDismiss={field.onBlur}
                 onIonChange={(e) => {
-                  // these two fields need to stay in sync
-                  setValue("metadata_submission.templates.0", e.detail.value);
-                  field.onChange(e.detail.value);
+                  // The `packageName` and `templates` fields need to stay in sync.
```

Comment:
As I am reading this (before having read the remainder of the diff), I am wondering whether the author meant to write `template`—instead of `templates`—in the comment.

_Edit: Nope! It's a reference to the `templates` (plural) property of the `metadata_submission` object. I'll resolve this conversation immediately._